examples/amp-photo-split-compare.amp.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
```

Comment:
I'd keep the example file in a separate PR and only merge when the extension is ready to launch. This is so we don't confuse other developers as to whether or not this has launched.